src/Compression/CompressionCodecDeflateQpl.h

```diff
@@ -86,6 +86,7 @@ class HardwareCodecDeflateQpl
     /// For flush, pop out job ID && job object from this map. Use job ID to release job lock and use job object to check job status till complete.
     std::map<UInt32, qpl_job *> decomp_async_job_map;
     Poco::Logger * log;
+    SoftwareCodecDeflateQpl &sw_codec;
```

Comment:
Cosmetic: `SoftwareCodecDeflateQpl & sw_codec;`

Reply:
Please add a comment behind `sw_codec` that sw_codec provides a fallback in case of errors.

Reply:
Done